src/app/core/services/issue.service.ts

```diff
@@ -395,6 +426,19 @@ export class IssueService {
     }
   }
 
+  private parseTesterResponse(toParse: string): TesterResponse[] {
+    let matches;
+    const testerResponses: TesterResponse[] = [];
+    const regex = /(## \d.*)[\r\n]*(.*)[\r\n]*(.*)[\r\n]*\*\*Reason for disagreement:\*\* ([\s\S]*?(?=-------------------))/gi;
+    while (matches = regex.exec(toParse)) {
+      if (matches && matches.length > 1) {
```

Comment:
1 here seems to be a magic number, maybe can replace with a variable to improve readability